interfaces/PA-App/src/app/personal-components/select-language/select-language.component.html

```diff
@@ -2,7 +2,10 @@
   <dialogue-turn>
     <p [innerHTML]="'personal.select-language.introduction'|translate"></p>
     <p>
-      <ion-button type="button" size="small">
+      <ion-button type="button" 
```

Comment:
Once we add a "MoreInfoButtonComponent", we should be able to include a `<more-info-button>`-element here, with an attribute of the translated message:

```
<more-info-button [message]="{{ 'personal.select-language.info-message'|translate }}">
  {{ 'personal.select-language.more-info'|translate }}
</more-info-button>
```
(So the more-info-button also contains the icon)

Reply:
The `translate`-magic should be, I think, some other syntax... But the main goal is to insert the 'already-translated-text' into the component, so it doesn't have to worry about translating it....

But on the other hand, it needs to load the TranslateService anyway to be able to translate the pop-up-title anyway... So might as well pass in only the 'key to be translated' (`personal.select-language.info-message`)